docs/LV_client/tools/mono_plot_radar_target_by_name.md

```diff
@@ -0,0 +1,33 @@
+# mono_plot_radar_target_by_name.vi
+
+<p class="img_container">
+<img class="lg_img" src="../mono_plot_radar_target_by_name.png"/>
+</p>
+
+### Description
+
+Tool used to generate plot to compare AoA, Range and Velocity between the GT targets and the target list from the Radar sensor. This tool only tracks one target by the specified target on the **Name of Target** control 
```

Comment:
Tool generates a plot to compare AoA, Range and Velocity between the GT targets and the target list from the Radar sensor. This tool only tracks one target by the specified target on the **Name of Target** control.